### PR TITLE
Feature - ok-to-test in github action

### PR DESCRIPTION
**Pull request by naveensrinivasan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
N/A

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Ok-to-test via github action based on this https://github.com/imjohnbo/ok-to-test

* **What is the current behavior?** (You can also link to an open issue here)
No e2e tests run for PR merges for forks

* **What is the new behavior (if this is a feature change)?**
Runs e2e in PR for git forks

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
None

* **Other information**:

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@inferno-chromium This is to test `ok-to-test` command. I am not sure whether this is going to work, but only after merging in, we can test it. 

I cannot test it on my fork. You can read more here https://github.com/imjohnbo/ok-to-test

**Comment by naveensrinivasan** [comment]
@inferno-chromium ping

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+name: Integration tests
+
+jobs:
+  integration-trusted:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'repository_dispatch') || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository)
+    steps:
+      - id: debug-outputs
+        run: |
+          echo ${{ github.event_name }}
```
Comment (inferno-chromium) [inline .github/workflows/integration.yml:16]: Do we need all these echos now?

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+          echo ${{ github.event.client_payload }}
+
+      - id: verify-sha-input
+        if: github.event_name == 'repository_dispatch'
+        run: |
+          echo \"${{ github.event.client_payload.pull_request.head.sha }}\"
```
Comment (inferno-chromium) [inline .github/workflows/integration.yml:22]: Do we need all these echos now?

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+          echo \"${{ github.event.client_payload.slash_command.args.named.sha }}\"
+          SHAINPUT=$(echo ${{github.event.client_payload.slash_command.args.named.sha}} | cut -c1-7)
+          if [ ${#SHAINPUT} -le 6 ]; then echo "error::input sha not at least 7 characters long" ; exit 1
+          else echo "done"
+          fi
+          SHAHEAD=$(echo ${{github.event.client_payload.pull_request.head.sha}} | cut -c1-7)
+          echo ${#SHAINPUT}
```
Comment (inferno-chromium) [inline .github/workflows/integration.yml:29]: Do we need all these echos?

File: .github/workflows/ok-to-test.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      with:
+        app_id: ${{ secrets.APP_ID }}
+        private_key: ${{ secrets.PRIVATE_KEY }}
+
+    - name: Slash Command Dispatch
+      uses: peter-evans/slash-command-dispatch@v1
+      env:
+        TOKEN: ${{ steps.generate_token.outputs.token }}
+      with:
+        #token: ${{ env.TOKEN }} # GitHub App installation access token
```
Comment (inferno-chromium) [inline .github/workflows/ok-to-test.yml:30]: can you remove this.

**Review by inferno-chromium — APPROVED** [review]

File: .github/workflows/integration.yml (hunk 1 above)
Comment (naveensrinivasan) [reply .github/workflows/integration.yml:16]: Echo's are needed for debugging. If it works the way we expect it to then I will do another PR to remove them all.

File: .github/workflows/ok-to-test.yml (hunk 4 above)
Comment (naveensrinivasan) [reply .github/workflows/ok-to-test.yml:30]: I can remove this.

File: .github/workflows/integration.yml (hunk 3 above)
Comment (inferno-chromium) [reply .github/workflows/integration.yml:29]: sounds good, perfect then!

File: .github/workflows/integration.yml (hunk 1 above)
Comment (inferno-chromium) [reply .github/workflows/integration.yml:16]: sounds good, perfect then!

**Pull request merged**